ignite-base/App/Config/ReduxPersist.js

```diff
@@ -3,10 +3,10 @@ import { AsyncStorage } from 'react-native'
 
 const REDUX_PERSIST = {
   active: true,
-  reducerVersion: '2',
+  reducerVersion: '4',
   storeConfig: {
     storage: AsyncStorage,
-    blacklist: ['login'], // reducer keys that you do NOT want stored to persistence here
+    blacklist: ['login', 'search'], // reducer keys that you do NOT want stored to persistence here
```

Comment:
I'm thinking we should switch to preferring `whitelist`.   Not relevant to this PR, but just thinking outloud.